pkgs/top-level/all-packages.nix

```diff
@@ -22495,7 +22496,13 @@ with pkgs;
   libiconv =
     if lib.elem stdenv.hostPlatform.libc [ "glibc" "musl" "nblibc" "wasilibc" "fblibc" ]
       then libcIconv (if stdenv.hostPlatform != stdenv.buildPlatform
-        then libcCross
+          then libcCross
+        else if stdenv.cc.nativeLibc
+          then {
+            pname = "libc-${stdenv.hostPlatform.system}";
+            version = "native";
+            dev = stdenv.cc.nativePrefix;
+          }
         else stdenv.cc.libc)
```

Comment:
Do we not have a stdenv.cc.libc on FreeBSD?

Reply:
This is another nativeLibc case. nativeLibc is where you're using nix on a non-nixos system and you're using the libc from that system. I originally added these lines for the original, impure bootstrap, which used that for the first phase. I can remove everything related to this if you want.